workflow-templates/executor-ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
             | sort -V | tail -1)
           pip install git+https://github.com/jina-ai/jina.git@v${JINA_VERSION}
       - name: Python requirements
-        run: pip install -r tests/requirements.txt
+        run: pip install -r requirements.txt -r tests/requirements.txt
```

Comment:
Some won't have core requirements, like SimpleIndexer. Should be optional, based on an if